templates/vertical-standard/page.html.hbs

```diff
@@ -1,54 +1,63 @@
 {{#> layouts/html }}
   {{#> script/core }}
     {{> cards/all }}
-    {{!-- {{> templates/vertical-standard/collapsible-filters/page-setup }} --}}
-    {{> templates/vertical-standard/script/appliedfilters }}
-    {{> templates/vertical-standard/script/verticalresultscount }}
-    {{> templates/vertical-standard/script/searchbar }}
-    {{> templates/vertical-standard/script/spellcheck }}
-    {{> templates/vertical-standard/script/navigation }}
-    {{> templates/vertical-standard/script/verticalresults }}
-    {{> templates/vertical-standard/script/pagination }}
-    {{> templates/vertical-standard/script/locationbias }}
-    {{!-- {{> templates/vertical-standard/script/sortoptions }} --}}
-    {{!-- {{> templates/vertical-standard/script/facets }} --}}
-    {{!-- {{> templates/vertical-standard/script/filterbox }} --}}
-    {{!-- {{> templates/vertical-standard/script/qasubmission }} --}}
+    {{!-- {{> templates/vertical-grid/collapsible-filters/page-setup }} --}}
+    {{> templates/vertical-grid/script/appliedfilters }}
```

Comment:
Was the change to use the `vertical-grid` templates here intentional? Each page template has its own set of partials in the `templates/{pageTemplate}` directory. The page template should use those partials, as well as some shared, common ones (such as `script/core`). If page templates refer to partials from other templates, that could make things difficult for the HHs. If a HH were to override one of the `templates/vertical-grid` partials, it would now impact both `vertical-grid` and `vertical-standard` pages. This would surprise them, as they'd expect just the pages built with the `vertical-grid` layout to change.

Reply:
sorry, it was a mishap switching between files for the templates. I updated and retested the changes!